File: semestre-1/algoritmos/MR-Alg-02/MR-Alg-02-Ex-07.py
quantDigitos = 3
minimo = 10 ** (quantDigitos - 1)
maximo = 10 ** (quantDigitos) - 1

n = int(input(f"Digite um número de {quantDigitos} digitos: "))
while not (minimo <= n <= maximo):
    print(f"O número precisa ter {quantDigitos} digitos!")
    n = int(input(f"Digite um número de {quantDigitos} digitos: "))

centena = n // 100
dezena = n // 10 % 10
unidade = n % 10

print(f"Centena: {centena}")
print(f"Dezena: {dezena}")
print(f"unidade: {unidade}")

File: semestre-1/algoritmos/MR-Alg-02/.c/MR-Alg-02-Ex-07.c
/*Dado um número de três algarismos N = CDU (onde C é o
algarismo das centenas, D é o algarismo das dezenas e U o algarismo das unidades) Faça um
programa Python que receba do usuário o número inteiro N, e imprima separadamente a
centena, a dezena e a unidade.*/

#include <stdio.h>

void main() {
    int n;
    printf("Digite um numero de 3 algarismos: ");
    scanf("%d", &n);

    int centena = n / 100;
    int dezena = n / 10 % 10;
    int unidade = n % 10;

    printf("Centena: %d\nDezena: %d\nUnidade: %d\n", centena, dezena, unidade);
}
